tests/Traits/DataHelpers/ConfigHelper.php

```diff
@@ -102,10 +103,32 @@ private function getDefaultFindologicConfigServiceMock(
                         $salesChannelId,
                         $languageId,
                         $config['integrationType']
+                    ],
+                    [
+                        'FinSearch.config.mainVariant',
+                        $salesChannelId,
+                        $languageId,
+                        $config['mainVariant']
                     ]
                 ]
             );
 
         return $configServiceMock;
     }
+
+    public function getFindologicConfig(array $override = [], bool $isDirectIntegration = true): Config
```

Comment:
Why does `$isDirectIntegration` default to `true`? Most of the time we want to ensure that the API behavior is the correct one, so it would make sense to default this to `false`.

Reply:
The default config we check for everywhere is based on Direct Integration being true, as even in the default config mock which we create (`\FINDOLOGIC\FinSearch\Tests\Traits\DataHelpers\ConfigHelper::getDefaultFindologicConfigServiceMock`), the default integration type is set as Direct Integration. So this was also done here in accordance with that.